cli/lib/kontena_cli.rb

```diff
@@ -87,6 +115,26 @@ def self.cli_root(*joinables)
       File.join(Kontena.root, 'lib/kontena/cli', *joinables)
     end
   end
+
+  def self.logger
+    return @logger if @logger
+    if log_target.respond_to?(:tty?) && log_target.tty?
+      logger = Logger.new(log_target)
+      logger.datetime_format = "%M%S.%2N "
```

Comment:
Sure, I agree that the timestamps for interactive `DEBUG=true` should be shorter than the default, and the PID isn't relevant here (the PID is relevant for e.g. the server logging, with multiple puma workers).

But `%M%S` isn't readable either. What is `4554.94`? Four and a half thousand seconds plus 940ms?